esphomeyaml/components/deep_sleep.rst

```diff
@@ -51,7 +51,24 @@ Configuration variables:
 
         mqtt:
           # ...
-          availability:
+          birth_message:
+            topic: sensor_name/status
+            payload: online
+          will_message:
+            topic: sensor_name/status
+            payload: online
```

Comment:
Instead of setting them to `online`, you should be able to set them both to ` ` (empty). Also, the note above should be updated too ("set `availability` to an empty value:").

Anyway, I already fixed this on the next branch for the next release.